EIPS/eip-2019.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: 2019
```

Comment:
2019 seems to be something entirely unrelated. EIP numbers are assigned based on issue or PR number and are not chosen freely. 2384 seems to be correct one for this.